sync/src/relayer/block_transactions_process.rs

```diff
@@ -32,13 +31,13 @@ impl<'a, CS: ChainStore + 'static> BlockTransactionsProcess<'a, CS> {
 
     pub fn execute(self) -> Result<(), FailureError> {
         let block_hash = cast!(self.message.block_hash())?.try_into()?;
-        if let Entry::Occupied(mut pending) = self
+        let mut success = false;
+        if let Some((compact_block, peers_set)) = self
             .relayer
             .shared()
             .pending_compact_blocks()
-            .entry(block_hash)
+            .get_mut(&block_hash)
```

Comment:
entry is better as before

https://github.com/nervosnetwork/lru-cache/blob/master/src/lib.rs#L132